examples/destination_iterable.tf

```diff
@@ -0,0 +1,78 @@
+resource "rudderstack_destination_iterable" "example" {
+  name = "my-iterable"
+
+  config {
+      api_key = "73983282843839749873"
+      
+      # map_to_single_event = true
+      # track_all_pages = false
+      # track_categorized_pages = true
+      # track_named_pages = true
+      # use_native_sdk {
+      #   web = true
+      # }
+      # initialisation_identifier { 
```

Comment:
should we use connection mode instead?

Reply:
We currently do not support new UI format in Iterable, it is still in old format.